### DISCUSSION TITLE
Timestamp showing as 2 hours earlier when using GraphQL

### DISCUSSION THREAD

**Discussion by arsekil** [keystonejs/keystone#8499] [Questions] [answered]
1. Create a Comment content in the Keystone Admin UI
2. Timestamp for Created At is automatically created correctly after my Laptops timezone.
3. When rendering that Comment content, it shows exactly 2 hours earlier than the it shows in the Admin UI.

I expecte that it reads the correct value and not alter it in any way.

![createdAt_timeError](https://user-images.githubusercontent.com/20362038/232246093-186c12eb-373c-4351-a5a3-1eda9902e841.png)
![createdAt_original](https://user-images.githubusercontent.com/20362038/232246097-e2aaadfc-ce89-49a2-a30b-7df3a56794bb.png)

My system is Node 18.15.0, npm 9,6,2, Win10Pro, latest stable version of keystone and DocumentRenderer

**Comment by borisno2** [comment]
Hi! I am interested to know how are you rendering the timestamp in your front end? Is "Created At" a separate field on the "Comment" list that is using the `timestamp` field with a default of `now`?

**Comment by arsekil** [comment]
Hi! @borisno2 

I had to realize, that it's not the DocumentRenderer that is rendering it incorrectly, so I changed the title of the issue as well.

But for context, I have a list called Comment, with 3 fields in it(text,document and timestamp). The timestamp has a defaultvalue of `now`.

I use GraphQL to return the data I need to render on the website and here's the snippet I'm using for the `createdAt` field:
```
Dátum: {v?.createdAt.slice(0,10)} {v?.createdAt.slice(11,19)}
```
where `v` is coming from the `myData.map((v,i) => { ... })` function I'm using to render all data coming from GraphQL. I'm using `.slice()`, because I want to omit the `T` from the timestamp that is returned.

I am sorry for the confusion about the title, it's not DocumentRenderer that's causing this.

**Comment by borisno2** [comment, marked as answer]
Hi @andrejmoltok 
The Datetime value that is returned in the GraphQL is in UTC, the AdminUI converts this to the local timezone of your browser, so in AdminUI you will always see local time. You will need to use javascript `Date` functions to convert the UTC time returned from GraphQL to local time.

**Reply by arsekil** [reply]
Thank you
